test/test_autograd.py

```diff
@@ -71,6 +71,9 @@ def getattr_qualified(obj, qname, default=None):
 
 PRECISION = 1e-4
 
+gradcheck = partial(gradcheck, check_batched_grad=True)
+gradgradcheck = partial(gradgradcheck, check_batched_grad=True)
```

Comment:
I was going to complain about how this is very unexpected but then I went and looked to see how many occurrences of gradcheck there are in this file and... well... there are a lot.